src/Network/Receive/fRO.pm

```diff
@@ -15,7 +15,7 @@
 package Network::Receive::fRO;
 
 use strict;
-use base 'Network::Receive::ServerType0';
+use base qw(Network::Receive::ServerType0);
```

Comment:
They're equivalent.

(But qw looks better, though)

Reply:
i use with use base 'Network::Receive::ServerType0'; not work for me !